kythe/cxx/indexer/cxx/GraphObserver.h

```diff
@@ -578,6 +578,13 @@ class GraphObserver {
   virtual void recordAbsVarNode(
       const NodeId& Node, const absl::optional<MarkedSource>& MarkedSource) {}
 
+  /// \brief Records a node representing a variable in a dependent type
```

Comment:
nit: C++ allows abstract variables, e.g.
```
template <typename T>
T pi = 3.14f;
```

So this won't necessarily be a "type" abstraction nor itself a type (for non-type template parameters).

Reply:
In `template <unsigned U> auto v = U;`, you can imagine the integer `u` substituted for `U` as `u` reified to a singleton type; C<1> and C<2> are not miscible because their types are different. I think it's valid to refer to tvars as variables at the type level.